modules/core/shared/src/main/scala/lucuma/core/enum/CatalogName.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.enum
+
+import lucuma.core.util.Enumerated
+
+sealed trait CatalogName extends Product with Serializable
+
+object CatalogName {
+  case object Simbad extends CatalogName
+  case object Horizon extends CatalogName
+  case object Gaia extends CatalogName
+
+  /** @group Typeclass Instances */
+  implicit val SectionVisibilityStateEnumerated: Enumerated[CatalogName] =
```

Comment:
`EnumeratedCatalogName`?

Reply:
I'm a little skeptical that this needs to be in the model since it's unused, and I'm not sure it would ever need to be in the database. Where do you see this going?